src/stories/Form.stories.tsx

```diff
@@ -41,16 +41,23 @@ const STORYBOOK_FORM_ID = 'TestForm' as keyof OnyxFormValuesMapping;
  *
  * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
  */
-const story: ComponentMeta<typeof FormProvider> = {
+const story: Meta<typeof FormProvider> = {
     title: 'Components/Form',
     component: FormProvider,
     subcomponents: {
+        // @ts-expect-error Subcomponent passes props with unknown type causing a TS error
```

Comment:
This is not a blocker or anything, just wondering why we need this.

Reply:
The following type error occurs without it:

<img width="1325" alt="Screenshot 2024-04-08 at 7 47 29 PM" src="https://github.com/Expensify/App/assets/30054992/3806ae9f-d381-4b15-82b6-21202797c381">

Reply:
Hehe yes, but why is the type wrong? Or is this expected somehow?

Reply:
I think there is something wrong indeed:

![image](https://github.com/Expensify/App/assets/6474442/5aef38de-7a1a-41bd-8477-c06793bd6c27)